[Fundamentals] Account security section (#4579)
cloudflare/cloudflare-docs: content/zaraz/get-started/_index.md

# Get started

> Before being able to use Zaraz, it is recommended that you proxy your website through Cloudflare. Refer to [Creating a Cloudflare account and adding a website](/fundamentals/get-started/setup/) for more information. If you do not want to proxy your website through Cloudflare, refer to [Use Zaraz on domains not proxied by Cloudflare](/zaraz/advanced/domains-not-proxied/).

Reviewer: ```suggestion
Before being able to use Zaraz, it is recommended that you proxy your website through Cloudflare. Refer to [Set up Cloudflare](/fundamentals/get-started/setup/) for more information. If you do not want to proxy your website through Cloudflare, refer to [Use Zaraz on domains not proxied by Cloudflare](/zaraz/advanced/domains-not-proxied/).
```
